cartographer_ros/configuration_files/revo_lds.lua

```diff
@@ -35,7 +35,9 @@ options = {
   lookup_transform_timeout_sec = 0.2,
   submap_publish_period_sec = 0.3,
```

Comment:
While we're here, maybe we could clarify the names of these two? `submap_list_publish_period_sec` and `transform_publish_period_sec`?

Reply:
+1 for `submap_list_publish_period_sec`. But `lookup_transform_timeout_sec` seems fine to me, it's used as a timeout for in the TF bridge, not for publishing. What you mean is probably `pose_publish_period_sec`, which is used for the `publish_local_trajectory_data_timer_` that publishes the TF.

Reply:
Yes, I was referring to `pose_publish_period_sec` -> `transform_publish_period_sec`.